packages/core/src/lib/implementation/report-to-gql.ts

```diff
@@ -69,6 +69,7 @@ function auditToGQL(audit: AuditReport): PortalAudit {
     formattedValue: audit.displayValue,
     ...(audit.details && {
       details: {
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
         ...(audit.details.issues && {
           issues: audit.details.issues.map(issueToGQL),
         }),
```

Comment:
While the `details` schema is [optional](https://github.com/code-pushup/cli/blob/main/packages/models/src/lib/audit-output.ts#L27), the `issues` property [is not](https://github.com/code-pushup/cli/blob/main/packages/models/src/lib/audit-output.ts#L8). Therefore, the optional object insertion should be removed and ESLint correctly assessed it as an unnecessary condition. Or am I missing something?

```suggestion
          issues: audit.details.issues.map(issueToGQL)
```